### ISSUE TITLE
June 2019 - Lesson release

### ISSUE THREAD

**Issue by fmichonneau** [carpentries/lesson-infrastructure#26]
Issue to keep track of the lessons interested in being released in June 2019

**Comment by kekoziar** [comment]
Regarding **File structure is clean (e.g. delete deprecated files, insure filenames are consistent)**, is there a list of depreciated files we should be on the lookout for?  I'm guessing the structure should match the [Carpentries/Styles](https://github.com/carpentries/styles)...?

**Comment by ErinBecker** [comment]
@kekoziar - that's a great question. In lessons I've worked on, there have often been old, unused data files, images, and episodes that have been replaced with new content but kept in the lesson repository. This checkbox is intended to prompt Maintainers to clean up any of those unused files. There may also be repos that have files that are no longer needed for our current template setup. For that, I'll defer to @fmichonneau's advice.
